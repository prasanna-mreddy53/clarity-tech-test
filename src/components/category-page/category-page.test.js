import React from "react";
import { shallow } from "enzyme";
import CategoryPage from "./category-page.component";
import store from "../../reducers/store";
import configureStore from "redux-mock-store";

const setup = (props) => {
  const component = shallow(<CategoryPage {...props} store={store} />);
  return component;
};
const buildStore = configureStore();

describe("Category Page Component Tests", () => {
  let categoryPage;
  let store;
  const initialState = {
    categories: {
      drinks: [
        {
          strCategory: "Ordinary Drink",
        },
      ],
    },
  };
  const props = {
    categories: [initialState],
  };
  beforeEach(() => {
    store = buildStore(initialState);
    categoryPage = setup(props);
  });

  it("renders correctly", () => {
    expect(categoryPage).toMatchSnapshot();
  });
});
