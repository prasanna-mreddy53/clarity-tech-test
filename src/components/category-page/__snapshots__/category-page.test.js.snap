// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category Page Component Tests renders correctly 1`] = `ShallowWrapper {}`;
